src/binary/ibc-relayer/commands/start.ts

```diff
@@ -262,5 +294,7 @@ async function relayerLoop(link: Link, options: Options, logger: Logger) {
     logger.info(`Sleeping ${options.poll} seconds...`);
     await sleep(options.poll * 1000);
     logger.info('... waking up and checking for packets!');
+
+    metrics?.pollCounter?.inc();
```

Comment:
With nullish coalescing operator, we can collect metrics nicely w/o condition blocks (if metrics are disabled).